pkg/operator/ceph/controller/spec.go

```diff
@@ -75,21 +75,64 @@ var (
 	cronLogRotate = `
 CEPH_CLIENT_ID=%s
 PERIODICITY=%s
-LOG_ROTATE_CEPH_FILE=/etc/logrotate.d/ceph
+LOG_ROTATE_CEPH_FOLDER=/etc/logrotate.d/ceph
+DAEMON_LOG_FILE=/var/log/ceph/
+LOG_MAX_SIZE=%s
+
+# size of log after removing any alphabet in the end. eg: for '1G' getting '1'
+LOG_MAX_SIZE_IN_DIGIT=$(echo "${LOG_MAX_SIZE}" | sed 's/.$//')
+# get last char of file size. eg: for '1G' getting 'G'
+GET_LAST_CHAR_OF_LOG_SIZE=$(echo $LOG_MAX_SIZE | rev | cut -c1)
+
+if [ "$GET_LAST_CHAR_OF_LOG_SIZE" == "G" ]; then
+	LOG_MAX_SIZE_IN_KB=$(( (LOG_MAX_SIZE_IN_DIGIT *1024*1024)))
```

Comment:
nit: don't need a space, right?
```suggestion
	LOG_MAX_SIZE_IN_KB=$(( (LOG_MAX_SIZE_IN_DIGIT*1024*1024)))
```